web/api/queries/stddev/stddev.c

```diff
@@ -120,6 +125,32 @@ calculated_number grouping_flush_coefficient_of_variation(RRDR *r, RRDR_VALUE_FL
     return  value;
 }
 
+calculated_number grouping_flush_zscore(RRDR *r, RRDR_VALUE_FLAGS *rrdr_value_options_ptr) {
+    struct grouping_stddev *g = (struct grouping_stddev *)r->internal.grouping_data;
+
+    calculated_number value;
+
+    if(likely(g->count > 1)) {
+        value = (lastv(g) - mean(g)) / stddev(g);
```

Comment:
If we have the possibility of `g` is only one value, the final result here will be `Inf`, are we sure we will have only vectors bigger than 1?

Reply:
I saw now that next lines has a test, please, ignore this.